app/models/developers/celebration_package_request.rb

```diff
@@ -1,8 +1,8 @@
-module Hired
-  class Form < ApplicationRecord
-    include Hired::Notifications
+module Developers
+  class CelebrationPackageRequest < ApplicationRecord
+    include Developers::CelebrationPackageRequests::Notifications
```

Comment:
```suggestion
    include Notifications
```

Reply:
Looks like in order to use bare `Notifications` include we'd need to drop the `s` from the `notifications.rb` which would break from naming convention of other `../../notifications.rb`

Should we rename `developers/celebration_package_requests/notifications.rb`?

Or keep `include Developers::CelebrationPackageRequests::Notifications` here?

Reply:
Ah, I see. Disregard my suggestion!

Reply:
Was able to drop the `Developers::` though! In 1ba19b2e416ac9013a9061e9ba4b76f167d1989f